asteriks_diamond_loop.c

```diff
@@ -0,0 +1,23 @@
+/*Ava N.*/
+/*This has half a diamond on request of the user's preference of length*/
```

Comment:
Hi Lesley, I have been working through the problem and I understand it now. Have a nice weekend and see you on Tuesday! 
Ava